bin/localeapp

```diff
@@ -68,6 +68,7 @@ command :install do |c|
     installer.config_type = :standalone if options[:standalone]
     installer.config_type = :heroku if options[:heroku]
     installer.config_type = :github if options[:github]
+    installer.write_dotenv = true if File.exist?('.env')
```

Comment:
Does not belong here, but to
`Localeapp::CLI::Install::DefaultInstaller` I think. Or better, maybe
a new installer, subclassing
`Localeapp::CLI::Install::DefaultInstaller`.

However in this file, we must add the command line switch to control
this new feature (whether the file should be created or not) explicitly.
IMHO, detecting file presence is a "bonus".